src/main/java/qna/domain/Answer.java

```diff
@@ -1,17 +1,36 @@
 package qna.domain;
 
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+import javax.persistence.Table;
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
 import java.util.Objects;
 
-public class Answer {
+@Entity
+@Table(name = "answer")
+public class Answer extends BaseEntity{
```

Comment:
```suggestion
public class Answer extends BaseEntity {
```
띄어쓰기 확인부탁드려요! IDE 자동줄맞춤 이용하시면 편리합니다!